src/lib/analysis_pipelines_config.tsv

```diff
@@ -0,0 +1,71 @@
+teamID	n_participants	excluded_participants	func_fwhm	con_fwhm
```

Comment:
This is the TSV version of this spreadsheet: https://docs.google.com/spreadsheets/d/1f-kBdWu5DKGFS4OLc32eHUxlvrof58nlP2NT5bL8jxE/edit#gid=1019165812